spec/jstests-spec.js

```diff
@@ -195,3 +195,75 @@ describe("Initializing tooltips", function() {
   });
 });
 
+describe("Pressing next and previous buttons associated with a graph", function() {
+  it("generates correct fetch request when next/previous buttons are pressed", function() {
+
+    const prevButton = document.createElement("button");
+    prevButton.id = "prevbutton";
+    prevButton.onclick = () => navigateGraph(-1);
+    const nextButton = document.createElement("button");
+    nextButton.id = "nextbutton";
+    nextButton.onclick = () => navigateGraph(1);
+    document.body.appendChild(prevButton);
+    document.body.appendChild(nextButton);
+
+    expect(currGraphNum).toBe(0);
+    expect(numMutations).toBe(3);
+
+    nextButton.click();
+    expect(currGraphNum).toBe(1);
+
+    nextButton.click();
+    expect(currGraphNum).toBe(2);
+
+    nextButton.click();
+    expect(currGraphNum).toBe(3);
```

Comment:
nextButton should already be disabled here? It might also make sense to assert that the buttons are not disabled in previous clicks.